index.yaml

```diff
@@ -5970,27 +5970,27 @@ entries:
   - annotations:
       catalog.cattle.io/certified: rancher
       catalog.cattle.io/display-name: Prometheus Federator
-      catalog.cattle.io/kube-version: '>= 1.16.0-0 < 1.28.0-0'
+      catalog.cattle.io/kube-version: '>= 1.23.0-0 < 1.29.0-0'
       catalog.cattle.io/namespace: cattle-monitoring-system
       catalog.cattle.io/os: linux,windows
       catalog.cattle.io/permits-os: linux,windows
       catalog.cattle.io/provides-gvr: helm.cattle.io.projecthelmchart/v1alpha1
       catalog.cattle.io/rancher-version: '>= 2.8.0-0 < 2.9.0-0'
       catalog.cattle.io/release-name: prometheus-federator
     apiVersion: v2
-    appVersion: 0.3.5
-    created: "2023-12-15T13:07:50.702182808-05:00"
+    appVersion: 0.3.4
+    created: "2024-04-04T14:30:03.632958-03:00"
     dependencies:
     - condition: helmProjectOperator.enabled
       name: helmProjectOperator
       repository: file://./charts/helmProjectOperator
     description: Prometheus Federator
-    digest: b78bd24740ed6de630e8db8366aada8928cb224eb580ee843b099a056ab33059
+    digest: 97740e89c6f45f34c08a6d9642b94cf80a49bc6cc402b57591e07a84fadc6fb5
     icon: https://raw.githubusercontent.com/rancher/prometheus-federator/main/assets/logos/prometheus-federator.svg
     name: prometheus-federator
     urls:
-    - assets/prometheus-federator/prometheus-federator-103.0.1+up0.4.1.tgz
-    version: 103.0.1+up0.4.1
+    - assets/prometheus-federator/prometheus-federator-103.0.1+up0.4.0.tgz
+    version: 103.0.1+up0.4.0
```

Comment:
This change looks suspect to me. Same thing for the lines above it in this file. Changes of this pattern are not present in any of the other forward port PRs I have seen. Are these changes supposed to be made?

Reply:
Agreed. Manually added the entry that was modified. Now, this only adds the entry for the new asset. Thanks for catching it